ci-operator/step-registry/hypershift/mce/kubevirt/conformance/hypershift-mce-kubevirt-conformance-workflow.yaml

```diff
@@ -0,0 +1,20 @@
+workflow:
+  as: hypershift-mce-kubevirt-conformance
+  documentation: |-
+    The HyperShift mce kubevirt conformance workflow executes tests against a new ephemeral
+  steps:
+    env:
+      COMPUTE_NODE_TYPE: c5n.metal
+      CONTROL_PLANE_INSTANCE_TYPE: m4.xlarge
+    post:
+    - chain: gather-core-dump
+    - chain: hypershift-mce-kubevirt-destroy-2.2
+    - chain: ipi-aws-post
+    test:
+    - chain: hypershift-conformance
+    pre:
+    - chain: ipi-aws-ovn-pre
+    - chain: ipi-install
+    - ref: hypershift-kubevirt-install
+    - ref: hypershift-mce-install
+    - chain: hypershift-mce-kubevirt-create-2.2
```

Comment:
Does this `hypershift-mce-kubevirt-create-2.2` step need to be tied directly to mce 2.2? what happens when we release and begin testing 2.3, 2.4, etc... Will we have to have a different set of steps for each mce version?

Is it possible that we simply have `hypershift-mce-kubevirt-create` and `hypershift-mce-kubevirt-destroy` steps which have the MCE_VERSION passed into them in order to account for any specific changes in behavior between the different mce versions in the future?

Reply:
Currently MCE 2.2 is still using the HyperShift CLI, in the future MCE 2.3 will use a different approach, considering the future scenario if 2.2 and 2.3 coexist, So we design it like this now, if we can easily integrate in the future, we will adjust our structure

Reply:
That's fine. If MCE 2.3 ends up being largely the same workflow, we can always change this later on.